### DISCUSSION TITLE
cool stuff

### DISCUSSION THREAD

**Discussion by nmsderp** [cytrinc/bark-coding#64] [Show and tell]
[recording.webm](https://github.com/mariocraft987/bark.github.io/assets/130254323/fc13350b-5727-4032-b3c3-6932227173b7)

**Comment by nmsderp** [comment]
https://github.com/mariocraft987/bark.github.io?tab=readme-ov-file#how-do-i-add-blocks

**Comment by raynepaws** [comment]
woah
